rocket_twin/systems/structure/nose.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from cosapp.base import System
+from OCC.Core.gp import gp_Pnt, gp_Vec
+from OCC.Core.TopoDS import TopoDS_Solid
+from pyoccad.create import CreateCone
+
+
+class Nose(System):
+    """A simple model of a nose.
+
+    Inputs
+    ------
+
+    Outputs
+    ------
+    rho [kg/m**3]: float,
+        density
+    shape: TopoDS_Solid,
+        pyoccad model
+    """
+
+    def setup(self):
+
+        # Geometric parameters
+        self.add_inward("radius", 1.0, desc="Base radius", unit="m")
+        self.add_inward("height", 1.0, desc="Height", unit="m")
+
+        # Positional parameters
+        self.add_inward("pos", 6.0, desc="Base center z-position", unit="m")
+
+        # Outputs
+        self.add_outward("shape", TopoDS_Solid(), desc="pyoccad model")
+        self.add_outward("rho", 3 / np.pi, desc="density", unit="kg/m**3")
```

Comment:
I understand why you put rho as an outward, but on the other hand... it should be an inward. 
What do you think about having rho as an inward, and compute weight as an outward here ?

Reply:
We could compute weight as an outward here instead of doing so at OCCGeometry. However, in order to compute the inertia tensor and the center of gravity, OCCGeometry needs the densities of each component